psalm.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
```

Comment:
This file is to be removed one https://github.com/Roave/infection-static-analysis-plugin/pull/248 is merged.

Reply:
Isn't that what `-c` is supposed to do within psalm?

Reply:
flags are completely ignored when using roave infection-static-analysis-plugin ( https://github.com/Roave/infection-static-analysis-plugin/issues/27 )

Reply:
Yes, see https://github.com/Roave/infection-static-analysis-plugin/pull/248#issuecomment-977702397 on the patch you proposed